lang/main-ru.json

```diff
@@ -920,6 +922,7 @@
         "roomname": "Укажите название комнаты",
         "roomnameHint": "Укажите название комнаты или ее адрес. Можете сами создать название и передать его будущим участникам встречи, чтобы они использовали именно его.",
         "sendFeedback": "Обратная связь",
+        "StartMeeting": "Создать конференцию",
```

Comment:
That should be `startMeeting`.

Reply:
Fixed by last commit df0b2b0045455cc3553ec0720985336d87bd3e2b . Also CLA was signed.